### ISSUE TITLE
[ACTIVE_PENDING_ARBITRATION] SWARM-1792 — GitHub-hosted LOCAL_FREE session slicing/resume V1

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1792]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
SWARM_WORKER_ID: SWARM-1792
SWARM_LANE_KEY: D08|LEARNED20M-GITHUB-HOSTED-LOCAL-FREE|REPRODUCIBILITY-RELEASE|SESSION-SLICING-RESUME-V1
PREFERRED_LANE: #9 D08 — Distributed Training / Cloud / Performance
PREFERRED_WORK_KIND: REPRODUCIBILITY_RELEASE
PARENT_ISSUES: #9 #16 #548 #653 #723
STATUS: ACTIVE_PENDING_POSTCLAIM_DIRECT_ARBITRATION
BASE_SHA: 2da11c62f066671a1122888ec116242502f4fb9c
CLAIMED_AT_UTC: THIS_ISSUE_UPDATED_AT_TIMESTAMP_IS_AUTHORITY
LEASE_UNTIL_UTC: CLAIM_UPDATED_AT_PLUS_6_HOURS
LOCAL_FREE_ONLY: true
PRODUCT_MUTATION: false
MERGE_AUTHORITY: false
TRAINING_CLAIM: false

OBJECTIVE:
Close the provider-execution gap between the existing canonical provider-neutral portable learned-20M run packet and the owner's GitHub-hosted-first execution policy. Implement a deterministic, fail-closed session-slicing/resume plan for standard free GitHub-hosted CPU jobs so a long lawful run can checkpoint with safety margin and resume in later jobs without restarting from zero. This package does NOT launch training and cannot widen any scientific or compute authority.

WHY THIS PACKAGE NOW:
- Highest data-spine blockers are actively owned (#1768 Rada->V9, #1764 PR1459 repair, current source executions); duplicating them would violate #723.
- Windows/operator preflight is live-owned by #1758; duplicate-proof launch lease/manifest is live-owned by #1782; both are explicit exclusions.
- PR #1514 resource-envelope package already has different-worker terminal PASS and awaits lawful integration; no duplicate audit.
- `configs/research/r01_portable_local_free_run_packet_v1.json` already requires session time limits, early checkpoints, atomic publish, fresh-process resume and cross-provider resume, but there is no dedicated GitHub-hosted session-slicing contract/PR in the preclaim semantic scan.
- The owner explicitly requires public-repo standard free GitHub-hosted runner first, benchmark/bounded pilot first, and exact checkpoint+resume across jobs when one job is insufficient.

OWNED_SURFACES ONLY AFTER ARBITRATION WIN:
- new D08 execution-profile/session-plan module consuming, not replacing, the existing portable-run-packet semantics;
- one machine-readable GitHub-hosted LOCAL_FREE CPU execution profile with conservative per-session wall-clock budget and checkpoint safety margin;
- deterministic slicing from total planned runtime / target work into ordered sessions, each with stop/checkpoint deadline and exact predecessor-resume identity requirements;
- fail-closed validator rejecting paid/non-free resource claims, over-budget sessions, missing content-addressed checkpoint identity, sequence gaps, stale/mismatched parent identity, and any claim that planning itself executed training;
- JSON/text CLI for PLAN and VALIDATE only; no workflow dispatch/trainer invocation;
- focused/adversarial tests and concise operator handoff documenting future canonical workflow integration.

AVOID_SURFACES:
- no PR #1336 trainer/runtime/optimizer/numerics mutation;
- no #1782 launch manifest/lease/idempotency implementation;
- no #1758 Windows preflight/safe-stop/operator UX implementation;
- no PR #1514 measured resource-envelope mutation;
- no corpus/dedup/decontamination/privacy/balance/split/tokenizer/packing/unique-loss/readiness/evaluation/final-test mutation;
- no GitHub Actions workflow creation or dispatch in this package;
- no integration/merge authority;
- no optimizer/model update, tokenizer fit, paid compute, foreign pretrained weights, external LLM/API corpus intelligence, or final-test access.

PACKAGE_DIMENSIONS:
- implementation_or_primary_research
- focused_tests
- adversarial_or_negative_tests
- machine_readable_evidence_or_validator
- documentation_or_operator_handoff
- live_authority_binding
- end_to_end_or_integration_proof
- reproducibility_proof

ACCEPTANCE:
1. Mandatory postclaim direct OPEN issue collection and separate OPEN PR collection are enumerated newest-first at 100/page through exact empty sentinels; any earlier still-valid same-semantic owner wins and SWARM-1792 yields before Product mutation.
2. Immediately before Product edits, late-bind live main and exact neighboring owner claims (#1758/#1782 plus any new successor); overlap => rotate.
3. Planner is deterministic and uses an explicitly conservative LOCAL_FREE GitHub-hosted CPU session cap/safety margin profile rather than assuming one unlimited job.
4. Multi-session plans preserve monotonic session indices and exact resume-parent checkpoint identity; a later slice cannot be validated without the immediately preceding authenticated checkpoint/result identity.
5. Validator rejects bool-as-int, nonfinite/negative durations, over-budget work windows, paid-resource widening, credential-bearing artifact URIs, sequence gaps, parent mismatch, and any positive `training_executed` / optimizer-update assertion in a planning artifact.
6. A one-session bounded pilot plan and a multi-session long-run plan both round-trip deterministically; serialization hash is stable.
7. Focused/adversarial tests pass locally; relevant wider repository gate passes where available. With GREEN CI pressure, open at most one PR after local proof.
8. Documentation makes clear this is execution planning only: launch still requires terminal corpus/tokenizer/exposure/runtime/recovery/lease authority, and actual GitHub workflow/trainer invocation remains a separate gated consumer.

FALLBACK:
Collision => mark ABANDONED_DUPLICATE and rotate on this same issue. If binding optimized-target exposure becomes positive and a lawful unowned training execution exists before Product mutation, release this lower-priority package and prioritize genuine bounded LOCAL_FREE training.

TRUTH_BOUNDARY:
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
TOKENIZER_FIT_AUTHORIZED=false
OPTIMIZER_UPDATES_EXECUTED_ON_REAL_TARGETS=0
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
FINAL_TEST_OUTCOMES_READ=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false
EXTERNAL_LLM_OR_API_USED_FOR_DATA_OR_INTELLIGENCE=false

NEXT NOW:
Mandatory direct OPEN issue + PR postclaim arbitration to exact empty sentinels. Only on WIN create `swarm/1792-github-hosted-session-plan` from exact live main and implement this bounded package.
